zebra-state/src/service.rs

```diff
@@ -202,7 +202,7 @@ impl StateService {
         let mut hashes = Vec::with_capacity(heights.len());
 
         for height in heights {
-            if let Some(hash) = self.hash(height) {
+            if let Some(hash) = self.best_hash(height) {
```

Comment:
good rename

Reply:
Yeah it was hard for coders to use the right functions, and reviewers to check they were using the right functions.